source/bdDetect.py

```diff
@@ -618,9 +625,13 @@ def addUsbDevices(self, type: DeviceType, ids: Set[str]):
 				f"Invalid IDs provided for driver {self._driver!r}, type {type!r}: "
 				f"{', '.join(malformedIds)}"
 			)
-		devs = self._getDriverDict()
-		driverUsb = devs[type]
-		driverUsb.update(ids)
+		
+		if not useAsFallBack:
+			devs = self._getDriverDict()
+			driverUsb = devs[type]
+			driverUsb.update(ids)
+		else:
+			DeviceMathcer.fallBackDevices.extend([(type, id) for id in ids])

```

Comment:
This looks like a typo to me
```suggestion
			DeviceMatcher.fallBackDevices.extend([(type, id) for id in ids])
```